bloom-housing/bloom
# 1691/hmi translations

# Pull Request Template

## Issue

Addresses #1691


## Description

When the HMI table was rewritten to include ranges I wrote those ranges as ($1,000 - $2,000) but it should have been ($1000 to $2000 per year). This is that bug fix and also another bug fix where the BMR headers were untranslated. 

## Type of change


## How Can This Be Tested/Reviewed?

Just start up the app and ensure all the seed data HMI tables are correct. The special cases are the BMR listing and both of the multiple AMI listings.

## Checklist:
